api/src/transactions.rs

```diff
@@ -1117,7 +1117,7 @@ impl TransactionsApi {
 
         // Simulate transaction
         let move_resolver = self.context.move_resolver_poem(&ledger_info)?;
-        let (status, output_ext) = AptosVM::simulate_signed_transaction(&txn, &move_resolver);
+        let (_, output_ext) = AptosVM::simulate_signed_transaction(&txn, &move_resolver);
```

Comment:
Could we get someone on the aptos VM to double check that it's supposed to come from the output @lightmark